daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -147,7 +146,10 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments) {
               }
             }
           }
-          case _ => Left("you should provide no more than one argument for this option") // Error because we expect there to be at most one flag
+          case _ =>
+            Left(
+              "you should provide no more than one argument for this option",
+            ) // Error because we expect there to be at most one flag
```

Comment:
> I've enabled rewrite.trailingCommas.style = always.

I suspect that scalafmt applies this parameter only to function call arguments, not method parameters, because this is one of the only few places I've seen a trailing comma added.  Oh well, if that's all scalafmt can do, that's okay.

This particular Left() call, BTW, was formatted oddly because there was a long comment after the right parenthesis.  This is one of the (I hope only a few) places where we're going to have to move such a comment to its own line to get the Left() back on one line, like this:

```scala
          case _ =>
            // Error because we expect there to be at most one flag
            Left("you should provide no more than one argument for this option")
```

Reply:
So, I think for this we need the following:

```
danglingParentheses.callSite = true
danglingParentheses.defnSite = true
rewrite.trailingCommas.style = always
```

I've put that in the config so we can look at that.

Reply:
Yes, v3 looks much better now!  Others, please take a look and review too.